docs/http/traffic-policy/expressions.mdx

```diff
@@ -1,14 +1,18 @@
+---
+pagination_label: Expressions
+---
+
 import ConfigExample from "/src/components/ConfigExample.tsx";
 
 # Expressions
 
-A policy is written with [Common Expression Language (CEL)](https://github.com/google/cel-spec) expressions
-to determine on which connections it is applicable. Each policy expression must evaluate to true in order for
-the policy's actions to take effect against a connection.
+Traffic Policy module enables you to filter inbound and outbound traffic with [Common Expression Language (CEL)](https://github.com/google/cel-spec) expressions. Each policy rule expression must evaluate to true in order for
+the rule's actions to take effect against traffic. In addition to CEL's [built-in functions and macros](https://github.com/google/cel-spec/blob/master/doc/langdef.md#macros) we provide additional variables for the [connection](#connection-variables), [request](#request-variables) and [response](#response-variables)
```

Comment:
Idk if "additional variables" makes sense here — we already say "In addition to CEL's built-in functions/macros" so maybe something like

>In addition to CEL's [built-in functions and macros](https://github.com/google/cel-spec/blob/master/doc/langdef.md#macros) we provide variables from the [connection](#connection-variables), [request](#request-variables) and [response](#response-variables) along with our own [custom macros](#macros).

? 🤷🏻‍♀️

Reply:
The change here also makes the paragraph under `#macros` a bit redundant too